monitoring-as-code/src/mac-config.libsonnet

```diff
@@ -16,7 +16,7 @@ local macDashboardPrefix = {
 // Dashboard documentation to be added to text panel. Content to be written as markdown.
 local dashboardDocs = {
   overView: '#### Overview\n\n**Purpose:** Observability of all products and tenants running on a platform.\n\n**Directions for use:** An aggregated SLO Status is presented in table panels by Product. The first table displays the aggregated SLO Status with supplementary data regarding traffic and alerts. The second table displays the SLO Status grouped by SLI type. The dashboard can be filtered by Environment.',
-  productView: '#### Product View\n\n**Purpose:** Observability of all the user journeys running on an individual product.\n\n**Directions for use:** Distinct user journeys are presented as row panels. SLIs are presented as stat panels with colour thresholds indicating the SLO Status. SLI Types (such as Availability and Latency) are aggregated into a single SLI Stat Panel. The dashboard can be filtered by Environment.',
+  productView: '#### Product View\n\n**Purpose:** Observability of all the user journeys running on an individual product.\n\n**Directions for use:** Distinct user journeys are presented as row panels. SLIs are presented as stat panels indicating the SLO Status, Error Budget and SLO target as set in the mixing defs. Colour thresholds have been set to highlight and identify issues. SLI Types (such as Availability and Latency) are aggregated into a single SLI Stat Panel. The dashboard can be filtered by Environment.',
```

Comment:
`defs` can we use the full term `definition file` please @georgeowusuHO just to avoid ambiguity.

Reply:
ok done